src/processes/scoring/calc-user-score.js

```diff
@@ -8,6 +8,7 @@ const {
   ageScore
 } = require('../../utils');
 const {calcKarmaScore} = require('./calc-karma-score');
+const UsersFunction = require('../../databases/functions/users');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `UsersFunction` is added correctly. However, it's a good practice to keep all the import/require statements at the top of the file. This helps in maintaining the code readability and understanding the dependencies at a glance.
```diff
+ const UsersFunction = require('../../databases/functions/users');
  ageScore
} = require('../../utils');
const {calcKarmaScore} = require('./calc-karma-score');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->